cmd/edge-proxy/main.go

```diff
@@ -137,12 +140,10 @@ func main() {
 				},
 			}, func(string, string) bool { return true }, func(ctx context.Context) error {
 				fmt.Printf("edge-proxy reverse tunnel established\n")
-
```

Comment:
Are these newline removals intentional?  Perhaps put them in a separate "whitespace fix" commit?

Reply:
Done.